src/devices/src/legacy/serial.rs

```diff
@@ -11,8 +11,8 @@ use std::os::unix::io::AsRawFd;
 
 use logger::{Metric, METRICS};
 use polly::event_manager::{EventManager, Subscriber};
-use utils::epoll::{EpollEvent, EventSet};
-use utils::eventfd::EventFd;
```

Comment:
Even though this appears to be imported from our `utils`, if you take a look here: https://github.com/firecracker-microvm/firecracker/blob/master/src/utils/src/lib.rs#L9, you can see that it actually belongs to `vmm_sys_util`. It's fine to use `utils` as an wrapper over `vmm_sys_util` since their purpose is kind of the same and `utils` basically extends `vmm_sys_util`'s number of utilities. These being said, if you just add `epoll` to the line that I mentioned above in the link, all the `use` changes shouldn't be needed anymore. Basically, you will need to do only the `&event` -> `event` changes.

Also, just realized now, the PR that added the `epoll` wrapper in `vmm_sys_util`, added also the `syscall.rs` file. It would be nice to use it from there and remove the file from `utils`. I will open an issue for that, feel free to do also this change in your PR if you want :D.

Reply:
As Laura also pointed out, re-exporting modules from vmm-sys-util as part of the `utils` crate was a (poorly explained :( ) by design decision.

What we were trying to achieve is having vmm-sys-util referenced as a dependency only once (when that is possible) because it is easier to do dependency control. When we want to update the vmm-sys-util version, we just have to update the dependency in a single Cargo.toml instead of chasing all of them throughout the code.

Reply:
Ah, I see. Thanks, I didn't realize that was the intent but makes sense. I'll adjust accordingly and include syscall as well.